tests/test_trigger_bisect_jobs.py

```diff
@@ -197,6 +200,13 @@ def test_problems():
     openqa.main(args)
     openqa.openqa_clone.assert_not_called()
 
+    args.url = "http://openqa.opensuse.org/tests/101"
+    try:
+        openqa.main(args)
+    except FileNotFoundError as e:
+        if str(e) == "[Errno 2] No such file or directory: 'tests/data/python-requests/openqa.opensuse.org/tests/101/investigation_ajax'":
```

Comment:
Since this is Python and not Perl we can likely do better than comparing the error message on string level. Maybe one can access the path somehow directly?